### PR TITLE
Adapts folder layout of config id specific result files

### PR DESCRIPTION
**Pull request by vulder**
resolves se-sic/VaRA#977

### CODE REVIEW

File: varats/varats/projects/test_projects/example_test_repo.py [hunk 1]
```diff
@@ -28,7 +28,7 @@ class ExampleTestRepo(VProject):
             project_name="example_test_repo",
             remote="BasicTestRepos/ExampleRepo",
             local="example_repo",
-            refspec="origin/HEAD",
+            refspec="HEAD",
```
Comment (boehmseb) [inline varats/varats/projects/test_projects/example_test_repo.py:31]: What's the reason for this change?

Comment (vulder) [reply varats/varats/projects/test_projects/example_test_repo.py:31]: It was wrong it the first place and does not work :wink:

**Review by boehmseb — APPROVED** [review]
